public/locales/en/translation.json

```diff
@@ -123,6 +123,7 @@
   "ConfirmationGoHome": "Go back to Home",
   "ConfirmationNotReceivedInstructions": "If you have not received a text message please proceed to the following steps:",
   "ConfirmationNotReceivedFirst": "If you have not received a text message, please resubmit your Name and Phone number below.",
+  "ConfirmationNotReceivedThird": "If you have tried previous steps and are still not receiving a text message, please contact us at 800-555-1234.",
```

Comment:
The phone number is now just the "tech4all@buildjustly.org" email. Other than that, it looks good to me!